modules/core/src/GVL.ts

```diff
@@ -474,7 +474,9 @@ export class GVL extends Cloneable<GVL> implements VendorList {
 
         try {
 
-          await this.fetchJson(url);
+          if (!skipFetchCall) {
+            await this.fetchJson(url);
+          }
```

Comment:
I think we could support this directly in the `GVL` constructor by giving the language, perhaps putting the lang as a parameter of an object to be given as a second parameter to the `GVL` constructor. In this way future additions will not cause breaking changes.

My opinion: Imagine calling the `GVL` constructor and then having to call this method with `skipFetchCall` set to `true`, it doesn't seem to fit well. Maybe you might like what I proposed. I hope this thought can help